docs-client/src/lib/transports/annotated-http.ts

```diff
@@ -112,16 +112,18 @@ export default class AnnotatedHttpTransport extends Transport {
       body: bodyJson,
     });
     const applicationType = httpResponse.headers.get('content-type') || '';
-    const response = await httpResponse.text();
-    if (response.length > 0) {
-      if (applicationType.indexOf('json') > -1) {
-        const prettified = prettify(response);
-        if (prettified.length === 0) {
-          return response;
-        }
+    if (applicationType.indexOf('json') > -1) {
+      const prettified = jsonPrettify(
+        JSON.stringify(await httpResponse.json()),
```

Comment:
Could you make sure this does not break the case where a user sends a property with a big integer? e.g.

```
{
  "big_int": 2934812984902318490
}
```

Could you make sure that:

- a response with a large value is parsed correctly and displayed in the response pane;
- a request with a large value is sent as-is to the server;
- a request with a large value is minified correctly into the browser location field; and
- the request shared via a URL is deserialized and prettified correctly into the debug form?

Reply:
We probably need front-end testing automation :sob:

Reply:
> Could you make sure this does not break the case where a user sends a property with a big integer? e.g.

Sorry, I missed this comment. 😭 

> We probably need front-end testing automation 😭

Indeed, we need it. 😭

Reply:
I didn't know that `JSON.parse("...")` does not properly handle a large value.
We might need to patch `jsonPrettify` to render Unicode.

Reply:
- [x] a response with a large value is parsed correctly and displayed in the response pane;
- [x] a request with a large value is sent as-is to the server;
- [x] a request with a large value is minified correctly into the browser location field; and
- [x] the request shared via a URL is deserialized and prettified correctly into the debug form?

I found out that [json-bigint](https://www.npmjs.com/package/json-bigint) keeps large numbers from truncating.
Finally, all problems are solved. 😀